src/Apps/CollectorProfile/Routes/MyCollection/CollectorProfileMyCollectionRoute.tsx

```diff
@@ -1,14 +1,3 @@
-import { Text } from "@artsy/palette"
-import { useFeatureFlag } from "System/useFeatureFlag"
+import { MyCollectionRouteRefetchContainer } from "Apps/Settings/Routes/MyCollection/MyCollectionRoute"
 
-const CollectorProfileMyCollectionRoute: React.FC = () => {
-  const isCollectorProfileEnabled = useFeatureFlag("cx-collector-profile")
-
-  if (!isCollectorProfileEnabled) {
-    return null
-  }
-
-  return <Text>Ja Hallo!</Text>
-}
-
-export const CollectorProfileMyCollectionRouteFragmentContainer = CollectorProfileMyCollectionRoute
+export const CollectorProfileMyCollectionRouteRefetchContainer = MyCollectionRouteRefetchContainer
```

Comment:
**question:** Can we just remove this file and import `MyCollectionRouteRefetchContainer` directly wherever it is needed?

Reply:
I'd keep this file just to make it clear that this is a different thing
When cleaning up the FF I'd remove this file and just move the MyCollectionRouteRefetchContainer here + renamed some variables, queries, etc

Reply:
Sounds good 👍  To be honest, I cannot tell what `CollectorProfileMyCollectionRoute` is supposed to be 🤔.

Reply:
MyCollection, but in the collector profile 😅

Reply:
collector-profile/my-collection

Reply:
Ooooh, I see :)